components/sysutils/nut/Makefile

```diff
@@ -101,27 +134,85 @@ CONFIGURE_OPTIONS +=	--with-net-snmp-config=$(USRBINDIR)/net-snmp-config-$(BITS)
 # Note: building the numerous documents of this projects is time-consuming
 # and we only need one copy. Also, we don't ship "dblatex" (needed here for
 # compiling PDFs) and probably don't require "html-chunked" chapters.
-CONFIGURE_OPTIONS +=        --with-doc="man=yes"
+CONFIGURE_OPTIONS +=	--with-doc="man=yes"
+
+# NUT-Monitor GUI client
+CONFIGURE_OPTIONS +=	--with-nut-monitor=yes
+CONFIGURE_OPTIONS +=	--with-pynut=yes
+CONFIGURE_OPTIONS +=	--with-python2=no
+CONFIGURE_OPTIONS +=	--with-python=python$(PYTHON_VERSION)
+
+# Be sure package is squeaky clean
+CONFIGURE_OPTIONS +=	--enable-Werror --enable-warnings --enable-Wcolor
+CONFIGURE_OPTIONS +=	--enable-keep_nut_report_feature
+CONFIGURE_OPTIONS +=	--enable-silent-rules
+
+# What do we run in case of `make check`?
+CONFIGURE_OPTIONS +=	--enable-spellcheck
+# NUT Integration Test suite, can take time and might fail due
+# to environment (busy ports, congested hsots and timeouts):
+#CONFIGURE_OPTIONS +=	--enable-check-NIT
+# No CPPCHECK here yet:
+CONFIGURE_OPTIONS +=	--disable-cppcheck
 
 CONFIGURE_OPTIONS +=	$(CONFIGURE_OPTIONS.$(BITS))
 
+# Fiddle with packaging:
+# * move man1m pages to target directory (see also patches/07-man.patch)
+# * move python stuff to vendor-packages
+# * make sure full python program name is in the shebang
+#   (and note we can not use ...\1${PYVER} because it is
+#    also numbers and SED gets confused about group \12 or so)
+# * pre-compile PYC files
+# * apply a little hack to comment away the import which
+#   confuses pkgdepend (it looks for "import" lines and
+#   disregards "try/except" around it)
+COMPONENT_POST_INSTALL_ACTION += \
+    $(MKDIR) $(PROTOUSRSHAREMANDIR)/man1m \
+        && $(MV) $(PROTOUSRSHAREMANDIR)/man1/*.1m $(PROTOUSRSHAREMANDIR)/man1m/ \
+        || { echo "FAILED to fix up man1m" >&2 ; exit 1; } ; \
+    for PYVER in $(PYTHON_VERSIONS) ; do \
+        $(MKDIR) "$(PROTOUSRLIBDIR)/python$${PYVER}/vendor-packages" \
+            && $(MV) "$(PROTOUSRLIBDIR)/python$${PYVER}/site-packages/"* "$(PROTOUSRLIBDIR)/python$${PYVER}/vendor-packages/" \
+            && find "$(PROTOUSRLIBDIR)/python$${PYVER}/vendor-packages/" -name '*.py' | while read F ; do \
+                $(GSED) '/^\(\#\!.*\/python\)$$/ s/$$/'"$${PYVER}"'/' -i "$$F" || exit ; \
+            done \
+        && python$${PYVER} -m compileall "$(PROTOUSRLIBDIR)/python$${PYVER}/vendor-packages" || exit ; \
+        case "$${PYVER}" in \
+        2.*) python$${PYVER} -c "import pygtk;pygtk.require('2.0');import pynotify" >/dev/null || { \
+                echo "Neutering import of pynotify in NUT-Monitor-py2gtk2 for this build with python$${PYVER}" >&2 ; \
+                $(GSED) -i "$(PROTOUSRSHAREDIR)/nut/nut-monitor/app/NUT-Monitor-py2gtk2" \
+                    -e 's,\(\s\)\(import pynotify\),\1return; \#\#\#NOT_HERE_\2,' || exit ; \
+            } ; \
+        esac ; \
+    done;
+
+test: $(TEST_64)
```

Comment:
Is this really needed?

Reply:
Not quite: I don't think it runs tests despite the presence (or absence) of this line. But generally, it would be nice to confirm that the in-distro build is valid. Not too hard a requirement for NUT though - its CI farm includes OI and OO.

Reply:
What is OO?

If the line is not needed, then please remove it.  If it is needed, then please add a comment above it why it is needed.  Thank you.